iaia/magicmodule.py

```diff
@@ -86,21 +107,32 @@ def make_prompt(self, *args, **kw):
 {source}"""
         return prompt, source
 
-    def get_completion(self, prompt, signature):
+    def get_completions(self, prompt, signature):
         response = gpt_client.create_completion(
             engine=self.module.gpt_engine,
             prompt=prompt,
-            max_tokens=1000,
-            temperature=0.1,
-            stop=["```"],
+            max_tokens=2000,
+            temperature=0,
+            stop=["``"],
```

Comment:
Ah yes, I encountered that too sometimes, it just not giving three backtics for some reason